### ISSUE TITLE
The Function name didn't display on the content panel when the "High Contrast" is on

### ISSUE THREAD

**Issue by yanlxu** [Microsoft/pxt-adafruit#618]
Repro steps:
• Navigate to https://makecode.adafruit.com/beta
• Click the "High Contrast On" under setting "More..."
• Click the "FUNCTIONS" in the editor
• Make a Function named "test"

**Expected result:**
1. It should display "Make a Function" when click the "FUNCTIONS", just like on **Edge&IE**
2. The Function name "test" should be displayed on the content panel

**Actual result:**
1. It didn't display "Make a Function" when click the "FUNCTIONS" on **Chrome&FireFox**
2. The Function name "test" didn't display on the content panel on all browser

**Chrome&FireFox:**
![function-chrome firefox](https://user-images.githubusercontent.com/11880996/35671744-44b55496-0734-11e8-8b74-bb54c7584097.png)

**Edge&IE:**
![function-edge ie](https://user-images.githubusercontent.com/11880996/35671766-53afa1e0-0734-11e8-9cca-ec33bafa1bba.png)

All browser:
![function](https://user-images.githubusercontent.com/11880996/35671817-7650b3a6-0734-11e8-9a53-1c6e42ed9a26.gif)

**Comment by samelhusseini** [comment]
Similar to the issue with the color field. We need to get rid of the tron look I think.

**Comment by samelhusseini** [comment]
Looks like a dupe of https://github.com/Microsoft/pxt-adafruit/issues/606
